reframe/frontend/cli.py

```diff
@@ -803,6 +815,10 @@ def _case_failed(t):
             list_checks(testcases, printer, options.list_detailed)
             sys.exit(0)
 
+        if options.list_tags:
+            list_tags(testcases, printer)
+            sys.exit(0)
+
```

Comment:
I think that you should add this action later on as well when we list them in case no action is specified.